pkg/cmd/roachtest/zip_util_test.go

```diff
@@ -21,64 +21,82 @@ import (
 )
 
 func TestMoveToZipArchive(t *testing.T) {
-	dir := t.TempDir()
-	p := func(elems ...string) string {
-		return filepath.Join(append([]string{dir}, elems...)...)
-	}
-	require.NoError(t, os.WriteFile(p("a1"), []byte("foo"), 0777))
-	require.NoError(t, os.WriteFile(p("a2"), []byte("foo"), 0777))
-	require.NoError(t, os.WriteFile(p("a3"), []byte("foo"), 0777))
-	require.NoError(t, os.Mkdir(p("dir1"), 0777))
-	require.NoError(t, os.WriteFile(p("dir1", "file1"), []byte("foo"), 0777))
-	require.NoError(t, os.WriteFile(p("dir1", "file2"), []byte("foo"), 0777))
-	require.NoError(t, os.Mkdir(p("dir2"), 0777))
-	require.NoError(t, os.WriteFile(p("dir2", "file1"), []byte("foo"), 0777))
-	require.NoError(t, os.WriteFile(p("dir2", "file2"), []byte("foo"), 0777))
+	basedir := t.TempDir()
```

Comment:
nit: `baseDir` instead?

Reply:
It was actually `baseDir` before :) I'm seen it both ways enough times that I'm totally indifferent.